src/index.ts

```diff
@@ -1099,6 +1184,36 @@ async function main() {
           run(() => mergeNeurons(args.sourceNeuronId, args.targetNeuronId))
         )
     )
+    .addCommand(
+      new Command("register-vote")
+        .description("Vote on a specific proposal.")
+        .requiredOption("--neuron-id <neuron-id>", "Neuron ID", tryParseBigInt)
+        .requiredOption(
+          "--proposal-id <proposal-id>",
+          "Proposal ID",
+          tryParseBigInt
+        )
+        .requiredOption("--vote <vote>", "Vote (1 - YES, 2 - NO)", tryParseInt)
+        .action((args) =>
+          run(() => registerVote(args.neuronId, args.proposalId, args.vote))
```

Comment:
would adding a `toLower/UpperCase` for the `args.vote` be handy?

Reply:
Vote I'm expecting a number. Let me improve the description, I understand the confusion.

Reply:
Is it clearer now, in the description it says "1 for YES, 2 for NO".